src/scripts/main.js

```diff
@@ -1,3 +1,26 @@
 'use strict';
 
-// write code here
+const list = document.querySelector('.tree');
+const listOfItems = document.querySelectorAll('li');
+
+listOfItems.forEach(item => {
+  const span = document.createElement('span');
+
+  item.prepend(span);
+  span.append(span.nextSibling);
+
+});
```

Comment:
Don't add extra line fro project 
```suggestion
  span.append(span.nextSibling);
});
```